src/EmployeeBook.java

```diff
@@ -0,0 +1,225 @@
+import java.util.Arrays;
+
+public class EmployeeBook {
+    private Employee[] employee = new Employee[10];
+
+
+    public void listAll(Employee[] arr) {
+        System.out.println(Arrays.toString(arr));
+    }
+
+    public void salary(Employee[] arr) {
+        double sum = 0;
+        for (int i = 0; i < arr.length; i++) {
+            sum += arr[i].getSalary();
+        }
+        System.out.println(sum);
+    }
+
+    public void minSalary(Employee[] arr) {
+        double min = 1000000;
+        for (int i = 0; i < arr.length; i++) {
+            min = arr[i].getSalary();
+            if (arr[i].getSalary() > arr[i + 1].getSalary()) {
+                min = arr[i + 1].getSalary();
+            }
+        }
```

Comment:
Немного подправлю алгоритм:
```suggestion
        double min = Double.MAX_VALUE;
        for (int i = 0; i < arr.length; i++) {
            if (min > arr[i].getSalary()) {
                min = arr[i].getSalary();
            }
        }
```

Аналогично поправь алгоритм максимума.
Ну и учти, что могут быть пустые значения в массиве.